core/services/chainlink/application.go

```diff
@@ -337,24 +321,20 @@ func NewApplication(config *orm.Config, ethClient eth.Client, advisoryLocker pos
 		subservices: subservices,
 	}
 
-	headTrackables = append(
-		headTrackables,
-		logBroadcaster,
-		ethConfirmer,
-		jobSubscriber,
-		pendingConnectionResumer,
-		balanceMonitor,
-		promReporter,
-		headBroadcaster,
-	)
+	headBroadcaster.SubscribeUntilClose(jobSubscriber)
```

Comment:
Why not subscribe the EthConfirmer here?

seems like we have an inconsistent pattern, because EthConfirmer subscribes itself but the headbroadcaster subscribes children here

Reply:
true, good point. I first wanted to have all of them subscribe themselves, but these cases were problematic. So yes can have all subscribed here